docs/whats-new/2020-12.md

```diff
@@ -0,0 +1,123 @@
+---
+title: ".NET docs: What's new for December 2020"
+description: "What's new in the .NET docs for December 2020."
+ms.date: 01/04/2021
+---
+
+# .NET docs: What's new for December 2020
+
+Welcome to what's new in the .NET docs for December 2020. This article lists some of the major changes to docs during this period.
+
+## .NET
```

Comment:
There isn't much for .NET and .NET fundamentals, we could manually combine those two in and .NET Core under one .NET heading

Reply:
Once I ship a new version of the what's new page generator tool, your repo's config file can be updated to automatically merge multiple areas/folders into 1 heading.